README.md

```diff
@@ -1,16 +1,156 @@
-# Title
+# Pseudocoding
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Work through the process of writing pseudocode
 
 ## Introduction
 
-## SWBAT 1
+Have you ever sat down to write something, such as an essay for school or an
+email to a coworker, and found that you needed to write a first draft before the
+final version that you turned in or sent? The first draft is a good opportunity
+to get your ideas all out without worrying about making them perfect from the
+beginning. You can do the same thing for code by writing a "first draft" using
+high-level language that describes what is happening but saves you from writing
+out the full code syntax. We call this _pseudocoding_.
 
-## SWBAT 2
+Pseudocoding is useful because it helps us make sure that we concentrate on the
+primary objectives of our code rather than the details or the format. This also
+aligns with our Flatiron Process, which reminds us to focus on small problems
+first and work our way up. So when we begin a new coding project, pseudocode is
+a great first step. Here, we'll work through the process of writing it.
+
+## Work Through the Process of Writing Pseudocode
+
+We're going to write code to create even chunks of delicious baguette for
+several people at our party. Baguettes are good to think about because they're
+basically rulers that you can eat. Here's a picture.
+
```

Comment:
I like the hand written pseudocode examples. I think if we are keeping them, we should maybe call out earlier on in this lesson that this process is something a student can do on paper as it is more about thinking of the process than typing the correct syntax. Maybe even encourage the reader to follow along on their own piece of paper. Would force them to look at each line of the pseudocode example